python/sglang/bench_serving.py

```diff
@@ -1524,6 +1555,8 @@ async def limited_request_func(request_func_input, pbar):
     print("{:<40} {:<10.2f}".format("P95 ITL (ms):", metrics.p95_itl_ms))
     print("{:<40} {:<10.2f}".format("P99 ITL (ms):", metrics.p99_itl_ms))
     print("{:<40} {:<10.2f}".format("Max ITL (ms):", metrics.max_itl_ms))
+    print("{s:{c}^{n}}".format(s="Cache Hit Rate", n=50, c="-"))
+    print("{:<40} {:<10.2f}".format("Cache Hit Rate:", metrics.cache_hit_rate))
```

Comment:
@hnyls2002 @zhyncs is it ok to include a new entry cache hit rate into the `bench_serving`? Please let us know if you don't want to change the displayed metrics.